GSA/px-bears-drupal
# [265] - Add a script to get db dumps and update readme

## PR Summary

This PR suggests the following changes:
1. adds a script to create a database dump,
2. includes direction to execute the script in the readme file,
3. deletes unnecessary script files.

## Related Github Issue

- fixes #[Create db backups for v2#265](https://github.com/GSA/px-bears-drupal/issues/265)

## Detailed Testing steps
1. Make sure you login cloud.gov on your terminal and target your org and space.

```
cf login -a api.fr.cloud.gov --sso
```

4. Change directory to where the script lives.

```
cd bin/cloudgov
```

5. Execute the following script.

```
./db-backup.sh
```
